test/api/v3/integration/groups/POST-groups_groupId_join.test.js

```diff
@@ -90,7 +115,7 @@ describe('POST /group/:groupId/join', () => {
     });
   });
 
-  context('Accepting invitation to a party', () => {
+  context('Joining a party', () => {
     let user, invitedUser, party;
```

Comment:
Judging by the travis CI and my local lint run it passes. Also this is not even part of the PR and is commited code, so it already passed it :P